### PR TITLE
Translation api

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/views/translation/database-tables/import.vue [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+      return {
+        options: {
+            autoStart:false,
+            target: (file,request,c)=>{
+                const urls = {
+                    'application/zip'  : `/translation-api/database-table/${this.$route.params.table}/import/zip`,
+                    'application/json' : `/translation-api/database-table/${this.$route.params.table}/import/json`
+                }
+                let url = urls[file.fileType];
+                url = `${url}/${this.fileLanguage?.code}`;
```
Comment (sbilodeau) [inline app/views/translation/database-tables/import.vue:174]: no encodeURIcomponent 
using ?. assum that the `fileLanguage` canc be `null` that will lead
to
`url/undefined`

File: package.json [hunk 2]
```diff
@@ -5,13 +5,14 @@
     "cheerio": "^1.0.0-rc.10",
     "easy-zip": "0.0.4",
     "ejs": "2.5.5",
+    "esm": "^3.2.25",
```
Comment (sbilodeau) [inline package.json:8]: you seems to have upgrade the project toe module. do you still need this module?

**Review by sbilodeau — CHANGES_REQUESTED** [review]
Little things to fix

File: app/views/translation/database-tables/import.vue (hunk 1 above)
Comment (blaisefonseca) [reply app/views/translation/database-tables/import.vue]: fixed

File: package.json (hunk 2 above)
Comment (blaisefonseca) [reply package.json:8]: no, fixed

File: app/views/translation/database-tables/import.vue [hunk 3]
```diff
@@ -171,7 +171,7 @@ export default {
                     'application/json' : `/translation-api/database-table/${this.$route.params.table}/import/json`
                 }
                 let url = urls[file.fileType];
-                url = `${url}/${this.fileLanguage?.code}`;
+                url = `${url}/${this.fileLanguage?.code||''}`;//language param is optional
```
Comment (sbilodeau) [inline app/views/translation/database-tables/import.vue:174]:
```suggestion
                url = `${url}/${encodeURIComponent(this.fileLanguage?.code||'')}`;//language param is optional
```

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**